app/models/ship.rb

```diff
@@ -0,0 +1,4 @@
+class Ship < ApplicationRecord
+  belongs_to :user
+  validates :name, :description, :price_per_day, :available_from, :available_to, presence: true
```

Comment:
After defining the categories we want to have, there should be a validation like this:
validates :category, inclusion: { in: %w(luxury casual big small medium slow fast)
A good thing would also be make it possible to select many categories for same ship

Reply:
For the :capacity attribute we will need a validation such as:
validates :capacity, length: { minimum: 4, maxium: 20 }